lib/evss/disability_compensation_form/form0781.rb

```diff
@@ -47,11 +77,7 @@ def create_form(incidents)
           'email' => @phone_email['emailAddress'],
           'veteranPhone' => @phone_email['primaryPhone'],
           'veteranSecondaryPhone' => '', # No secondary phone available in 526 PreFill
-          'veteranServiceNumber' => '', # No veteran service number available in 526 PreFill
-          'incidents' => incidents,
-          'remarks' => @form_content['remarks'],
-          'additionalIncidentText' => @form_content['additionalIncidentText'],
-          'otherInformation' => @form_content['otherInformation']
+          'veteranServiceNumber' => '' # No veteran service number available in 526 PreFill
```

Comment:
I am not sure if this is a number we have somewhere (or can get via api call). If it is not, are we asking them for it? Or just leaving it blank on the form? Is it on the form?

Reply:
No change was made here from how 0781 translation currently is configured and documented. There is a service number field on both the current and new form. Would you propose a change be made to obtain this value?

Reply:
oh I see, I didnt see that it was already up there and was just moved, it looked like you had added it. My bad. No I dont think, if we dont ask for it, then maybe we dont need to? 

Might be a questions for Ruben? IE "hey there is this service number and Secondary phone number on the form, we dont collect that, should we? Do we need to? etc

idk that this is blocking merge of this though! but we should find out.